### DISCUSSION TITLE
Workflow notification mail template

### DISCUSSION THREAD

**Discussion by brianteeman** [joomla/joomla-cms#47076] [3. Approved Features]
All components that send emails now use the mail templates component now except for workflow notifications where you still customise the message as plain text in the component and have no further templating options.

Well except that this is not exactly true. By accident I noticed that you can customise the email as it is using the Messages: New message template even though that us **_Email containing a message created in the messaging component._**

![image](https://user-images.githubusercontent.com/1296369/166885245-c017784e-9959-4105-bc0b-0fd8debfaca0.png)

I would have expected it to have its own mail template something like user action logs

![image](https://user-images.githubusercontent.com/1296369/166885378-b9ecaf77-d8d3-418a-b4a9-a41c1f5603fb.png)

# Approval Information
Approved: 18.03.2026
Minimum Target Version: 6.2

**Comment by HLeithner** [comment]
**The Joomla Maintainer Team decided at the 18.03.2025 meeting to accept this feature.**
